iocore/dns/CMakeLists.txt

```diff
@@ -15,24 +15,19 @@
 #
 #######################
 
-
-add_library(inkdns STATIC
-    DNS.cc
-    DNSConnection.cc
-    Inline.cc
-    SplitDNS.cc
+add_library(inkdns STATIC DNS.cc DNSConnection.cc Inline.cc SplitDNS.cc)
+target_include_directories(
+    inkdns
+    PRIVATE ${CMAKE_SOURCE_DIR}/iocore/eventsystem
+            ${CMAKE_SOURCE_DIR}/iocore/dns
+            ${CMAKE_SOURCE_DIR}/iocore/io_uring
+            ${CMAKE_SOURCE_DIR}/iocore/aio
+            ${CMAKE_SOURCE_DIR}/iocore/net
+            ${CMAKE_SOURCE_DIR}/iocore/cache
+            ${CMAKE_SOURCE_DIR}/iocore/hostdb
+            ${CMAKE_SOURCE_DIR}/proxy
+            ${CMAKE_SOURCE_DIR}/proxy/http
+            ${CMAKE_SOURCE_DIR}/proxy/hdrs
+            ${CMAKE_SOURCE_DIR}/mgmt
+            ${CMAKE_SOURCE_DIR}/mgmt/utils
 )
-target_include_directories(inkdns PRIVATE
-        ${CMAKE_SOURCE_DIR}/iocore/eventsystem
-        ${CMAKE_SOURCE_DIR}/iocore/dns
-        ${CMAKE_SOURCE_DIR}/iocore/io_uring
-        ${CMAKE_SOURCE_DIR}/iocore/aio
-        ${CMAKE_SOURCE_DIR}/iocore/net
-        ${CMAKE_SOURCE_DIR}/iocore/cache
-        ${CMAKE_SOURCE_DIR}/iocore/hostdb
-        ${CMAKE_SOURCE_DIR}/proxy
-        ${CMAKE_SOURCE_DIR}/proxy/http
-        ${CMAKE_SOURCE_DIR}/proxy/hdrs
-        ${CMAKE_SOURCE_DIR}/mgmt
-        ${CMAKE_SOURCE_DIR}/mgmt/utils
-)
```

Comment:
Does `cmake-format` have a option like `InsertNewlineAtEOF` of `clang-format`?

Reply:
Not that I could see :/